### PR TITLE
Send error messages from apollo to snackbar

### PR DESCRIPTION
**Pull request by drshrey**

### CODE REVIEW

**Comment by drshrey** [comment]
cdsyptqmjx

File: src/lib/apollo.js [hunk 1]
```diff
@@ -42,12 +43,10 @@ export default (GRAPHQL_URI = process.env.REACT_APP_CIRCUIT_URI + '/query') => {
   const errorLink = onError(({ graphQLErrors, networkError }) => {
     if (graphQLErrors)
       graphQLErrors.map(({ message, locations, path }) =>  
-        console.log(
```
[the remark this answers is not in the document]
Comment (drshrey) [reply src/lib/apollo.js]: yup, agreed, will change. i haven't gone through all components, will see how that holds up there

**Comment by aballman** [comment]
secrets-error
screen shot 2018-11-21 at 1 06 17 pm

These errors are expected. If we queried the individual secrets and there was no secret value that should be reported as an error. In this case we are polling for all of the global secrets so any that are missing values will report an error.

It's a bit odd because the error doesn't make sense as presented in other situations (as opposed to the project creation error message)

**Comment by aballman** [comment]
Fixed in https://github.com/codeamp/panel/pull/292. Closing.

**Pull request closed without merging**